Microsoft.AspNetCore.SystemWebAdapters.sln

```diff
@@ -59,6 +59,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Samples.MVCApp.Tests", "tes
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Samples.RemoteAuth.Forms.Tests", "test\Samples.RemoteAuth.Forms.Tests\Samples.RemoteAuth.Forms.Tests.csproj", "{7C994084-D425-487C-9266-413BB0C22695}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.SystemWebAdapters.Services", "src\Services\Microsoft.AspNetCore.SystemWebAdapters.Services.csproj", "{632E6195-4304-4C67-AABB-7CFC3F9086B6}"
```

Comment:
I don't love the Microsoft.AspNetCore.SystemWebAdapters.Services name. It sounds like something that someone might use on its own (with .FrameworkServices and .CoreServices providing some TFM-specific helpers).

Maybe `Microsoft.AspNetCore.SystemWebAdapters.Common` or `Microsoft.AspNetCore.SystemWebAdapters.Abstractions` would better indicate that this is a shared support package that most users won't be depending on directly except for interfaces and things like that?

Reply:
I've renamed to abstractions based on feedback from the standup